app/javascript/components/dropdowns/reputation/ReputationIcon.tsx

```diff
@@ -23,7 +23,7 @@ export const ReputationIcon = forwardRef<
         <Icon icon="reputation" alt="Reputation" />
         <span>{reputation}</span>
       </div>
-      {isSeen ? null : <div className="--notification" />}
+      <div className={'--notification ' + (isSeen ? '' : 'unseen')} />
```

Comment:
@kntsoriano I've changed this so it doesn't replace the element, just the class, which means we can use a CSS transition